in_toto/model.go

```diff
@@ -985,10 +987,16 @@ type Statement struct {
 	Predicate interface{} `json:"predicate"`
 }
 
-// ProvenanceStatement is the definition for an entire provenance statement.
+// ProvenanceStatement_SLSA01 is the definition for an entire provenance statement with SLSA 0.1 predicate.
+type ProvenanceStatement_SLSA01 struct {
```

Comment:
underscores are actually not recommended in Golang. I think I would go for ProvenanceStatementSLSA01 instead.